src/EnergyPlus/ScheduleManager.hh

```diff
@@ -68,7 +68,28 @@ struct EnergyPlusData;
 
 namespace ScheduleManager {
 
-    int constexpr MaxDayTypes(12);
+    enum class DayType
```

Comment:
New DayType enum in ScheduleManager that includes all of the standard day types (replaces one that was in WeatherManager that only had Sunday-Saturday. Has a dummy slot at zero to keep the daytype integers the same as before.